cfme/tests/intelligence/test_chargeback.py

```diff
@@ -36,7 +36,7 @@ def test_add_new_compute_chargeback():
 def test_compute_chargeback_duplicate_disallowed():
     ccb = new_compute_rate()
     ccb.create()
-    with error.expected('Name already exists'):  # currently cfme just blows up
+    with error.expected('Description has already been taken'):  # currently cfme just blows up
```

Comment:
That inline comment doesn't make sense to me. Was it saying that CFME currently blows up when we expect 'Name already exists' and now the comment can go away, or is it saying that regardless of the expected error CFME blows up? Or something else entirely?